liberty-maven-webapp-parent/pom.xml

```diff
@@ -50,14 +50,12 @@
                             <goals>
                                 <goal>stop-server</goal>
                             </goals>
-                            <configuration>
-                                <failOnError>false</failOnError>
-                            </configuration>
                         </execution>
                         <execution>
                             <id>create-server</id>
                             <phase>prepare-package</phase>
                             <goals>
+                                <goal>install-server</goal>
```

Comment:
So if both goals are called the file will get generated twice?  Is WDT listening to updates to that file?  If so, will that cause churn?  Rajiv was going to try your new code for a separate goal.  What happened there?